backend/bin/deal-observer-backend.js

```diff
@@ -103,10 +100,10 @@ const sparkApiSubmitDealsLoop = async (pgPool, { sparkApiBaseUrl, sparkApiToken,
       Sentry.captureException(e)
     }
     const dt = Date.now() - start
-    console.log(`Loop "${LOOP_NAME}" took ${dt}ms`)
+    console.log(`Loop "Submit deals to spark-api" took ${dt}ms`)
 
     if (INFLUXDB_TOKEN) {
-      recordTelemetry(`loop_${slug(LOOP_NAME, '_')}`, point => {
+      recordTelemetry('loop_submit_deals_to_sparkapi', point => {
```

Comment:
```suggestion
      recordTelemetry('loop_submit_deals_to_spark_api', point => {
```

(unless you want to preserve the point name)

Reply:
that was the plan. Do you think it's worth it to make a breaking change in the point name?

Reply:
The whole point of the PR is to keep existing telemetry names in tact, so I'll keep this one as well